project/src/components/map/map.tsx

```diff
@@ -6,7 +6,7 @@ import useMap from '../../hooks/useMap';
 import { Offer } from '../../types/offer';
 
 type MapProps = {
-  offers: Offer[],
+  offers: Offer[] | undefined,
```

Comment:
```suggestion
  offers: Offer[],
```
хммм, тут что-то не то